src/commands/codepush/release-electron.ts

```diff
@@ -37,11 +37,16 @@ export default class CodePushReleaseElectronCommand extends CodePushReleaseComma
   @hasArg
   public entryFile: string;
 
-  @help("Sourcemap filename of the resulting bundle. If omitted, a sourcemap will not be generated")
+  @help("Path to where the sourcemap for the resulting bundle should be written. If omitted, a sourcemap will not be generated")
```

Comment:
I think this wording could be confused with the `sourcemap-output-dir` parameter (it's not clear that this defines the name of the file rather than the directory).

I suggest:
> Filename (including path) for the sourcemap of the resulting bundle. If \"sourcemap-output\" and \"sourcemap-output-dir\" are omitted, a sourcemap will not be generated

Is there a particular impetus behind the `longName` change? I wonder if we should include a depreciated `sourcemapFilename` which gets mapped to `sourcemapOutput` with a warning?